concepts.tex

```diff
@@ -43,15 +43,21 @@
 An expression is \techterm{equality preserving} if, given equal inputs, the expression results in
 equal outputs. The inputs to an expression \oldtxt{is}\newtxt{are} the set of the expression's operands. The
 output of an expression is the expression's result and all operands modified by the expression.
-\enternote Not all input values must be valid for a given expression; e.g., for integers \tcode{a}
+
+\pnum
+Not all input values must be valid for a given expression; e.g., for integers \tcode{a}
 and \tcode{b}, the expression \tcode{a / b} is not well-defined when \tcode{b} is \tcode{0}. This
-does not preclude the expression \tcode{a / b} being equality preserving. \exitnote
+does not preclude the expression \tcode{a / b} being equality preserving. \newtxt{The
+\techterm{definition space} of an expression is the set of input values for which the
+expression is well-defined.}
```

Comment:
The current standard already uses the term "*the domain of*". Perhaps we should bless that as a Phrase of Power.